src/modules/esl-animate/README.md

```diff
@@ -4,58 +4,36 @@ Version: *1.0.0*.
 
 Authors: *Anna-Mariia Petryk*, *Alexey Stsefanovich (ala'n)*, *Julia Murashko*.
 
-**_Important Notice: the component is under beta version, it is tested and ready to use but be aware of its potential critical API changes._**
-
 <a name="intro"></a>
 
-**ESLAnimate** is a module to animate items on viewport intersecting.
+## ESLAnimate
 
-Features:
-- Add class(es) when observed elements enter viewport area
-- Support group animation that allows item delay its animation on the passed time after previously animated item
-- Support forward and backward animations directions
-- Support automatic re-animate after item exit viewport area
-- JS API (Service) + Custom Element - Plugin for simple initialization
+**ESLAnimate** is a custom element module used to animate items on viewport intersection, which automatically initializes `ESLAnimateService` from html.
 
-## ESLAnimateService
-ESLAnimateService is a core of esl-animate module. Element needs to be observed by ESLAnimateService 
-in order to be animated.
-
-### Service API:
```

Comment:
Original order is more correct 
Service is the main feature
Element + Mixin are "sugar"